db_config.py

```diff
@@ -0,0 +1,60 @@
+import psycopg2
+
+db_url = "dbname='sendit_db' user='postgres' password='postgres' host='localhost' port='5432'"
```

Comment:
You never want to store sensitive information, like your database password, in your source code. If your source code is ever seen by anyone, they now have access to your database.
You should conceal/hide your production password in an environment variable.

Also create three databases, a test db, a development db and a production db, best without a password (I prefer you only set up a password for your production db).

Then configure your local machine to work with the development db, like this `export FLASK_ENV=development`

Reply:
Okay, let me see how I can do this